src/modules/select/wallets/wallet-link.ts

```diff
@@ -17,7 +17,7 @@ function walletLink(
   } = options
 
   return {
-    name: label || 'WalletLink',
```

Comment:
@vishnumad Just checking, is this intentional to change the WalletLink label to `Coinbase Wallet`?

Reply:
Yup, it's intentional. Since Coinbase Wallet is the only wallet that integrates with WalletLink, it's a little more clear to users that clicking on that will connect them with their Coinbase Wallet.